angular-cli-build.js

```diff
@@ -16,7 +16,8 @@ module.exports = function(defaults) {
       'reflect-metadata/**/*.+(ts|js|js.map)',
       'rxjs/**/*.+(js|js.map)',
       '@angular/**/*.+(js|js.map)',
-      '@angular2-material/**/*.js'
+      '@angular2-material/**/*.js',
+      'lodash/lodash.js'
```

Comment:
mmm not sure that lodash is mandatory here :/
Will attempt to live without it on this project :p